presto-main/src/main/java/com/facebook/presto/FullConnectorSession.java

```diff
@@ -109,6 +114,12 @@ public long getStartTime()
     }
 
     @Override
+    public <T> T getSystemProperty(String name, Class<T> type)
+    {
+       return type.cast(systemProperties.get(name));
```

Comment:
Thanks. I thought I had used the change but might have somehow missed it. It is using the sessionPropertyManager now.